requirements.txt

```diff
@@ -14,7 +14,7 @@ simplejson>=3.10.0,<4.0.0
 scikit-image>=0.14.0,<0.17.0
 scikit-build<1.0.0
 statsmodels==0.9.0
-simpleitk<2.0.0
```

Comment:
What changed with SimpleITK >2.0.0 ?

Do we know if this will break functionality of our stuff which uses SimpleITK?